frontend/2014-kashmir-floods-explorer/index.html

```
@@ -0,0 +1,397 @@
+    <section class="flood-hero">
+        <div class="hero-content">
+            <div class="hero-badges">
+                <span class="badge ramsar"><i class="fa-solid fa-water"></i> 2014 Floods</span>
+                <span class="badge himalayan"><i class="fa-solid fa-mountain-sun"></i> Kashmir Valley</span>
+                <span class="badge severe"><i class="fa-solid fa-exclamation-triangle"></i> Major Disaster</span>
+            </div>
+            <h1 class="hero-title animate-slide-up" style="color: white; font-size: 3.5rem;">2014 Jammu & Kashmir Floods</h1>
+            <p class="hero-subtitle animate-fade-in-delayed" style="color: var(--saffron); font-size: 1.5rem; font-family: 'Playfair Display', serif; margin-bottom: 15px;">One of India's worst natural disasters in recent history</p>
+            <p class="hero-location" style="color: #eee; font-size: 1.1rem; margin-bottom: 25px;"><i class="fa-solid fa-map-pin"></i> Kashmir Valley, Jammu & Kashmir, India</p>
+            <div class="hero-actions">
+                <a "#timeline" class="btn btn-primary ripple">Event Timeline</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)